frontend/ui/wikipedia.lua

```diff
@@ -828,6 +884,8 @@ function Wikipedia:createEpubWithUI(epub_path, page, lang, result_callback)
             local _coroutine = coroutine.running()
             cur_progress_box = ConfirmBox:new{
                 text = text,
+                ok_text = _("Yes"),
```

Comment:
Is there something more descriptive that could go here?

Reply:
This function ui_progress_callback(text, confirmbox) is some kind of unaware-of-the-context function. It is supposed to replace a simple logger when using UI, so logs are displayed as InfoMessage, with an additional feature to get an answered choice via a ConfirmBox if confirmbox=true (thus, all the coroutine stuff to preempt UI).
It was previously the classic Cancel/OK, but for the existing text, they all goes well with Yes/No:
```
include_images = progress_callback(T(_("The page contains %1 images.\nWould you like to download and include them in the generated EPUB file?"), #images), true)
use_img_2x = progress_callback(_("Would you like to use slightly higher quality images? This will result in a bigger file size."), true)
go_on = progress_callback(T(_("Downloading image %1 failed. Continue anyway?"), inum), true)
if progress_callback(_("Download did not complete.\nDo you still want to create an EPUB with what has been downloaded so far?"), true) then
```
I could pass the ok_text and cancel_text as 3rd and 4th argument to progress_callback, but I would then use "Yes" and "No" everywhere :)

Reply:
No rewording suggestion for this last one ? :"Download did not complete.\nDo you still want to create an EPUB with what has been downloaded so far?"

Reply:
It's only about images, right?

Do you want to create an EPUB with the already downloaded images?

> I could pass the ok_text and cancel_text as 3rd and 4th argument to progress_callback, but I would then use "Yes" and "No" everywhere :)

Wouldn't it be something like
1. include
2. higher quality
3. continue
4. create

Reply:
Yes, except edge case when all images failed downloading :)

Not so sure about these contextual ok_test/cancel_text (and you gave me only the ok_text :) Somehow, It's more obvious with answering Yes/No to the asked questions (which are formulated expecting a Yes/No), rather than having to think (yes, a small thinking needed :) about how these no yes-no buttons answer the question.
Anyway, you tell me, but gimme cancel_text too, and the right capitalisation for button text please :)

Reply:
Don't include / Include
Standard quality / Higher quality
Stop / Continue
Don't create / Create